src/main/java/seedu/address/model/good/Good.java

```diff
@@ -19,11 +20,20 @@
     // Identity fields
     private final GoodName goodName;
     private final GoodQuantity goodQuantity;
+    private final GoodQuantity threshold;
 
     public Good(GoodName goodName, GoodQuantity goodQuantity) {
         requireAllNonNull(goodName, goodQuantity);
         this.goodName = goodName;
         this.goodQuantity = goodQuantity;
+        this.threshold = new GoodQuantity(DEFAULT_QUANTITY);
+    }
+
+    public Good(GoodName goodName, GoodQuantity goodQuantity, GoodQuantity threshold) {
+        requireAllNonNull(goodName, goodQuantity, threshold);
+        this.goodName = goodName;
+        this.goodQuantity = goodQuantity;
+        this.threshold = threshold;
     }
```

Comment:
There's overlap here, is it possible for one method to just pass to the other?

Reply:
ok